### PR TITLE
pistachio: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
DEVICE_TITLE is split up into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT

### CODE REVIEW

**Comment by mwarning** [comment]
@adrianschmutzler

File: target/linux/pistachio/image/Makefile [hunk 1]
```diff
@@ -24,11 +24,12 @@ define Device/Default
 endef
 
 define Device/marduk
+  DEVICE_VENDOR := Creator
+  DEVICE_MODEL := Ci40
```
Comment (adrianschmutzler) [inline target/linux/pistachio/image/Makefile]: Looks right, but why is the block called "marduk"?

Comment (mwarning) [reply target/linux/pistachio/image/Makefile]: 'Creator Ci40 (marduk)'  0d271cef8071d0698246b4b50a610b5ff718af01

**Comment by adrianschmutzler** [comment]
http://lkml.iu.edu/hypermail/linux/kernel/1610.1/00737.html

> Add support for Imagination Technologies' Marduk board which is based
> on Pistachio SoC. It is also known as Creator Ci40.

So, sounds like one could add 
DEVICE_ALT0_VENDOR := Imagination Technologies
DEVICE_ALT0_MODEL := Marduk board

but I'm not sure whether we should

**Comment by mwarning** [comment]
@adrianschmutzler fine with me.

File: target/linux/pistachio/image/Makefile (hunk 1 above)
Comment (dangowrt) [reply target/linux/pistachio/image/Makefile]:
```
DEVICE_VENDOR:=Imagination Technologies
DEVICE_MODEL:=Creator Ci40 (VL-62899)
```

**Comment by mwarning** [comment]
@adrianschmutzler I've added your suggestions.

**Comment by hauke** [comment]
Thank you for your patch, it was applied to master.

**Pull request closed without merging**